go.mod

```diff
@@ -50,3 +50,5 @@ require (
 )
 
 replace github.com/takama/daemon v0.12.0 => github.com/codenotary/daemon v0.0.0-20200507161650-3d4bcb5230f4
+
+replace github.com/spf13/afero => github.com/spf13/afero v1.5.1
```

Comment:
just wondering if this is needed due to a change in this PR

Reply:
THis is needed to resolve go mod tidy issue